### ISSUE TITLE
We should use UINT32 for labels

### ISSUE THREAD

**Issue by haesleinhuepf** [clEsperanto/CLIc#244]
... e.g. in `masked_voronoi_labeling_func` and `extend_labeling_via_voronoi_func`. So this:

```
  tier0::create_like(src, flip, dType::FLOAT);
  tier0::create_like(src, flop, dType::FLOAT);

```

should be changed to this:

```
  tier0::create_like(src, flip, dType::UINT32);
  tier0::create_like(src, flop, dType::UINT32);

```

**Comment by StRigaud** [comment]
after  #243 is solve by the  #257 PR, rely on placeholder dtype:

```
tier0::create_like(src, flip, dType::LABEL);
tier0::create_like(src, flop, dType::BINARY);
```

**Comment by StRigaud** [comment]
closed with #257
